app/v1/controllers/requestParsers/validators/SubmitUkPropertyBsasValidator.scala

```diff
@@ -0,0 +1,118 @@
+          AdjustmentRangeValidation.validate(data.nonFurnishedHolidayLet.flatMap(_.expenses.map(_.other)).flatten, "other"),
+          AdjustmentRangeValidation.validate(data.nonFurnishedHolidayLet.flatMap(_.expenses.map(_.consolidatedExpenses)).flatten, "consolidatedExpenses")
+        )
+      case data if data.furnishedHolidayLet.isDefined =>
+        List(
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.income.map(_.rentIncome)).flatten, "rentIncome"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.premisesRunningCosts)).flatten, "premisesRunningCosts"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.repairsAndMaintenance)).flatten, "repairsAndMaintenance"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.financialCosts)).flatten, "financialCosts"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.professionalFees)).flatten, "professionalFees"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.costOfServices)).flatten, "costOfServices"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.travelCosts)).flatten, "travelCosts"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.other)).flatten, "other"),
+          AdjustmentValueValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.consolidatedExpenses)).flatten, "consolidatedExpenses"),
+
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.income.map(_.rentIncome)).flatten, "rentIncome"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.premisesRunningCosts)).flatten, "premisesRunningCosts"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.repairsAndMaintenance)).flatten, "repairsAndMaintenance"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.financialCosts)).flatten, "financialCosts"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.professionalFees)).flatten, "professionalFees"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.costOfServices)).flatten, "costOfServices"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.travelCosts)).flatten, "travelCosts"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.other)).flatten, "other"),
+          AdjustmentRangeValidation.validate(data.furnishedHolidayLet.flatMap(_.expenses.map(_.consolidatedExpenses)).flatten, "consolidatedExpenses")
+        )
+      case _ => List()
```

Comment:
This line isn't hit by coverage. Would it be worth adding a test for if both `furnishedHolidayLet` and `nonFurnishedHolidayLet` are undefined?